### PR TITLE
Update data-science-tutorial.md

### PR DESCRIPTION
**Pull request by KrisvanderMast**
The original code provided a message that it was obsolete since beginning 2021 and provided a replacement statement which I made up my edit with. However it turns out that accuracy became way less (0.61). Best to have it peer examined.

### CODE REVIEW

**Comment by KrisvanderMast** [comment]
Please cross reference with the expected outcome.

File: docs/datascience/data-science-tutorial.md [hunk 1]
```diff
@@ -247,7 +247,7 @@ A neural network is a model that uses weights and activation functions, modeling
 1. With the model built and trained its now time to see how it performs against the test data.
 
    ```python
-   y_pred = model.predict_classes(X_test)
+   y_pred = np.argmax(model.predict(X_test), axis=-1)
    print(metrics.accuracy_score(y_test, y_pred))
```
Comment (rohrlaf) [inline docs/datascience/data-science-tutorial.md:251]: I receive the same results with Keras [version `2.6.0`](https://github.com/keras-team/keras/wiki/Keras-2.0-release-notes) and they highly differ from the tutorial results (~80% vs. ~61%).

Python code:
```
   y_pred = np.argmax(model.predict(X_test), axis=1)
   print(metrics.accuracy_score(y_test, y_pred))
   print(metrics.classification_report(y_test, y_pred))
```

Output:
```
0.6124401913875598
              precision    recall  f1-score   support

           0       0.61      1.00      0.76       128
           1       0.00      0.00      0.00        81

    accuracy                           0.61       209
   macro avg       0.31      0.50      0.38       209
weighted avg       0.38      0.61      0.47       209
```

**Comment by jmew** [comment]
So it looks like the code (specifically the API predict_classes()) was deprecated in 2021, after this tutorial was written. As a short-term solution, I think we can accept this PR, and just update the rest of the rest of the tutorial to say 61% rather than 80%. In the long term, I think it might be best to find a better example/model for this tutorial (given how fast the data science space evolves).

**Review by Cheybaby95 — APPROVED** [review]

File: docs/datascience/data-science-tutorial.md (hunk 1 above)
Comment (minhhieuuuuu88) [reply docs/datascience/data-science-tutorial.md:251]: Tốt

> `

Comment (minhhieuuuuu88) [reply docs/datascience/data-science-tutorial.md:251]: Tốt

**Comment by rjmolyneaux** [comment]
Updating the docs to reflect current content. @KrisvanderMast Thank you for bringing this to our attention!

**Pull request merged**